docs/getting_started.md

```diff
@@ -268,14 +287,20 @@ to deploy to, but we never deployed that either! Let's go ahead and fix these as
    bicep.)
 
 1. Update the `ContainerRegistry.Tests.ps1` file to point to your new resource group:
```

Comment:
Can you add a note to the reader to take note of the output from the `New-AzResourceGroupDeployment` call. The output from that has the final container registry name.

Can you also update this line to something like "Update the `ContainerRegistry.Tests.ps1` file to replace the placeholder values with the final values:" or something similar.